index.js

```diff
@@ -6,9 +6,7 @@ module.exports = function () {
     return function () {};
   }
   var active = document.activeElement;
-  var ranges = Array.apply(Array, {
-    length: selection.rangeCount
-  }).map(function(range, index) {
+  var ranges = Array.apply(Array, [selection.rangeCount]).map(function(range, index) {
```

Comment:
@nkbt: I'm confused about the screenshot you posted. If you `console.log(i.length)` you'll see that they're both 5, Chrome just happens to log them differently.

In reality, this likely doesn't need to use `apply`, and could just use the `Array` constructor directly, but there's really no reason to `join()` and `split()` on the same string back to back...